pytest.ini

```diff
@@ -3,7 +3,11 @@
 #
 # xrootdpyfs is free software; you can redistribute it and/or modify it under
 # the terms of the Revised BSD License; see LICENSE file for more details.
+#
+#
+# addopts = --isort --pydocstyle --pycodestyle --doctest-glob="*.rst" --doctest-modules --cov=xrootdpyfs --cov-report=term-missing
```

Comment:
To be re-added and fixed in the next task: the upgrade of the module to the latest deps (pytest-invenio, code format, etc...)